example/yarn.lock

```diff
@@ -3398,9 +3398,9 @@ available-typed-arrays@^1.0.5:
   integrity sha512-DMD0KiN46eipeziST1LPP/STfDU0sufISXmjSgvVsoU2tqxctQeASejWcfNtxYKqETM1UxQ8sp2OrSBWpHY6sw==
 
 axios@^1.3.2:
-  version "1.3.5"
-  resolved "https://registry.yarnpkg.com/axios/-/axios-1.3.5.tgz#e07209b39a0d11848e3e341fa087acd71dadc542"
-  integrity sha512-glL/PvG/E+xCWwV8S6nCHcrfg1exGx7vxyUIivIA1iL7BIh6bePylCfVHwp6k13ao7SATxB6imau2kqY+I67kw==
+  version "1.3.2"
+  resolved "https://registry.yarnpkg.com/axios/-/axios-1.3.2.tgz#7ac517f0fa3ec46e0e636223fd973713a09c72b3"
+  integrity sha512-1M3O703bYqYuPhbHeya5bnhpYVsDDRyQSabNja04mZtboLNSuZ4YrltestrLXfHgmzua4TpUqRiVKbiQuo2epw==
```

Comment:
This mismatch, between the root and example yarn.lock files, produced a TypeScript error deep in `MockEnvironmentDecorator.tsx` that `yarn test` would catch on.  As you can see in package.json, I've added `tsc --noEmit` to the `test:ci` command so we can catch this at PR time.

Reply:
Scratch that; due to the WIP [here](https://github.com/lifeomic/react-native-sdk/blob/master/tsconfig.build.json#L17), we're not ready to `tsc --noEmit` as part of `test:ci`.  We'll get there in time.